datahub-web-react/src/app/SearchRoutes.tsx

```diff
@@ -58,8 +61,16 @@ export const SearchRoutes = (): JSX.Element => {
     const showTags =
         config?.featureFlags?.showManageTags &&
         (me.platformPrivileges?.manageTags || me.platformPrivileges?.viewManageTags);
-
+    
     const showIngestV2 = config.featureFlags.showIngestionPageRedesign;
+  
+    if (me.platformPrivileges === undefined) {
+        return (
+            <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', height: '100vh' }}>
```

Comment:
We can break this into a small component called ProtectedRouteLoadingIndicator.tsx?

And we generally try to avoid inline styling in this way. We can move this to a wrapper styled component!